test/core-meta.html

```diff
@@ -0,0 +1,163 @@
+<!doctype html>
+<!--
+Copyright (c) 2015 The Polymer Project Authors. All rights reserved.
+This code may only be used under the BSD style license found at http://polymer.github.io/LICENSE.txt
+The complete set of authors may be found at http://polymer.github.io/AUTHORS.txt
+The complete set of contributors may be found at http://polymer.github.io/CONTRIBUTORS.txt
+Code distributed by Google as part of the polymer project is also
+subject to an additional IP rights grant found at http://polymer.github.io/PATENTS.txt
+-->
+
+<html>
+<head>
+
+  <title>core-meta</title>
+  <meta charset="utf-8">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+
+  <script src="../../webcomponentsjs/webcomponents-lite.js"></script>
+  <script src="../../web-component-tester/browser.js"></script>
+  <script src="../../simple-fixture/simple-fixture-mocha.js"></script>
+
+  <link rel="import" href="../core-meta.html">
+  <link rel="import" href="../../simple-fixture/simple-fixture.html">
+
+</head>
+<body>
+
+  <simple-fixture id="TrivialMeta">
+    <template>
+      <core-meta id="info"></core-meta>
+    </template>
+  </simple-fixture>
+
+  <simple-fixture id="ManyMetas">
+    <template>
+      <core-meta id="default1"></core-meta>
+      <core-meta id="default2"></core-meta>
+      <core-meta id="default3"></core-meta>
```

Comment:
What is expected to happen if two or more `core-meta`s are created with the same ID? That isn't covered in the tests.